app/src/main/java/com/sun/uefascore/screen/fixtures/FixturesFragment.kt

```diff
@@ -17,6 +28,21 @@ class FixturesFragment : Fragment() {
         return inflater.inflate(R.layout.fragment_fixtures, container, false)
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initPresenter()
+    }
+
+    private fun initPresenter() {
+        val date = Date()
+        val formatter = SimpleDateFormat(Constant.DAY_FORMAT)
+        val dayDevice: String = formatter.format(date)
```

Comment:
biến địa phương không cần khai báo kiểu 
```suggestion
        val dayDevice = formatter.format(date)
```